doc/guides/security-release-process.md

```diff
@@ -38,6 +38,12 @@ information described.
   * Described in the pre/post announcements
 
 * [ ] Pre-release announcement [email][]: ***LINK TO EMAIL***
+  ```
```

Comment:
```suggestion
  ```text
```
Lint.

Reply:
Thanks, and sorry about the later response!